src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { EmptyPeopleList } from '../EmptyPeopleList/EmptyPeopleList';
+import { ErrorMessage } from '../ErrorMessage/ErrorMessage';
+import { PeopleList } from '../PeopleList/PeopleList';
+
+const peopleWithParents = (people: Person[]) => {
+  return people.map(person => {
+    const mother = people.find(pers => pers.name === person.motherName);
+    const father = people.find(pers => pers.name === person.fatherName);
+
+    return { ...person, mother, father };
+  });
+};
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loader, setLoader] = useState(false);
```

Comment:
Try to set `true` as the default value for `loader` to prevent showing `<EmptyPeopleList />` component after the page render.